app/models/speaker_email_template.rb

```diff
@@ -0,0 +1,5 @@
+class SpeakerEmailTemplate
```

Comment:
Couple suggestions here: One, a list defining email template types would be more at home in the Event model, since it refers to attributes on that model. Second, I would break this into two pieces: a simple array of valid template types, then a mapping between those types and their display text. 

There's a good example in models/proposal/state.rb. The basic types are defined as constants, then grouped into collections. The type-to-text mapping can live in that helper you made.